src/components/Modal/Modal.module.css

```diff
@@ -155,10 +155,6 @@
       display: flex;
       flex-direction: row-reverse;
       gap: var(--spacing-gap-sm, 8px);
-
-      :global(*) {
-        margin: 0 !important;
-      }
```

Comment:
Did you build and link this repository on the website so that other things wouldn't break?

Just use the instead of the dependency version inside the packages the words: 
`“@mia-platform-internal/console-design-system-react": "link../path/on/your/pc”`
`design system local repo - yarn build`

Reply:
I'd give special attention to Modal button featuring shortcuts. I guess this margin was set for some reason, I agree with removing it and fix the specific behaviour it was supposed to fix

Reply:
When you are ready and you tell me that everything is merging